### ISSUE TITLE
Previous weights are deleted when running transfer again

### ISSUE THREAD

**Issue by calaveraDeluxe** [matthew-sochor-zz/transfer#10]
After running transfer, a message says that
```
To further refine the model, run again with:

    transfer --run --project cats_vs_dogs
```
Doing this deletes the whole weights folder (the relevant line is [here](https://github.com/matthew-sochor/transfer/blob/master/transfer/model.py#L74)). Transfer then goes on to complain that the previous best model can not be found.

Unless i'm not understanding something correctly, I would say that the weights folder should not be deleted.

**Comment by matthew-sochor-zz** [comment]
Thanks for pointing this out (and sorry for the slow response!)  This used to be the behavior but I think I accidentally refactored it out at one point.  I'm going to look into it, as I would also like the ability to re-load a model and train it on new data from that point forward.

**Comment by matthew-sochor-zz** [comment]
@calaveraDeluxe should be fixed in https://github.com/matthew-sochor/transfer/commit/3dcdf00b0ced61502ccc3d2648b34de888845253

This is included in the latest pip install 0.40 if you update.  Please let me know if it worked, and thanks for the issue!

**Comment by matthew-sochor-zz** [comment]
anyway, pretty sure this is fixed, closing
